v2/tools/generator/internal/armconversion/convert_from_arm_function_builder.go

```diff
@@ -177,6 +178,18 @@ func (builder *convertFromARMBuilder) namePropertyHandler(
 		setAzureName), true
 }
 
+func (builder *convertFromARMBuilder) userAssignedIdentitiesPropertyHandler(
+	toProp *astmodel.PropertyDefinition,
+	_ *astmodel.ObjectType,
+) ([]dst.Stmt, bool) {
+	if _, ok := astmodel.IsUserAssignedIdentityProperty(toProp); !ok {
+		return nil, false
+	}
+
+	// TODO: For now we are not assigning these, as we don't know how to rebuild the reference
+	return nil, true
```

Comment:
Still true?

Reply:
Yes, this is still true. Related to #2829, this other direction is actually not used.